### ISSUE TITLE
False positive MD053 for image inside link

### ISSUE THREAD

**Issue by janosh** [DavidAnson/vscode-markdownlint#249]
I'm seeing a false-positive when wrapping referenced image inside a link:

```md
[![Binder]](test)

[Binder]: https://mybinder.org/badge_logo.svg
```

 ![Screen Shot 2022-08-31 at 20 57 03](https://user-images.githubusercontent.com/30958850/187828593-721d54ab-fb8b-4930-b137-adf5c8550445.png)
